src/bubble_sort.c

```diff
@@ -0,0 +1,43 @@
+#include <stdbool.h>
+#include <stdio.h>
+
+#define SIZEOF(arr) sizeof(arr) / sizeof(*arr)
+
+void swap(int* xp, int* yp) {
+    int temp = *xp;
+    *xp = *yp;
+    *yp = temp;
+}
+
+void printArray(int* arr, int size) {
+	int i;	
+
+	for (i = 0; i < size; i++) {
+		printf("%d ", arr[i]);
+	}
+
+	printf("\n");
+}
+
+int main() {
+	int arr[] = {10, 11, 5, 2, 4, 0, 11};
+	int arrSize = SIZEOF(arr);	
+	
+	int i, j;
+	bool swapped;
+
+	for(i = 0; i < arrSize; i++) {
```

Comment:
Extract the sorting part into separate function

Reply:
done